src/transformers/modeling_tf_utils.py

```diff
@@ -642,8 +642,9 @@ def generate(
                 `What are attention masks? <../glossary.html#attention-mask>`__
 
             decoder_start_token_id=None: (`optional`) int
-                If an encoder-decoder model starts decoding with a different token than BOS.
-                Defaults to `None` and is changed to `BOS` later.
+                Start token id for the decoder. Defaults to `decoder_start_token_id` as defined the model's config or to the `bos_token_id`
```

Comment:
The content between single backquotes will render in italics in the doc. It's ok for "optional" but  in most places, argument names are rendered as objects, so I think you should use ` ``decoder_start_token_id`` ` or ` :obj:`decoder_start_token_id` ` and same for bos_token_id